core/consensus/sumeragi.cpp

```diff
@@ -159,7 +157,7 @@ namespace sumeragi {
         std::int64_t    commitedCount = 0;
         std::uint64_t   numValidatingPeers;
         std::string     myPublicKey;
-        std::deque<std::unique_ptr<peer::Node>> validatingPeers;
+        std::vector<std::shared_ptr<peer::Node>> validatingPeers;
```

Comment:
`sumeragi` doesn't has to have peerList. It is enough to use `validatinIps`( that is gotten `getIpList()` ).
But, It occures copy of string. It is worse efficiency than `getPeerList` after fixed.